Dockerfile.windows.x86_64

```diff
@@ -0,0 +1,32 @@
+# escape=`
+
+ARG WIN_VER="ltsc2016"
+
+FROM mcr.microsoft.com/windows/servercore:$WIN_VER
+
+ADD https://aka.ms/vs/16/release/vs_buildtools.exe C:\TEMP\vs_buildtools.exe
+ADD https://win.rustup.rs/x86_64 C:\TEMP\rustup-init.exe
+ADD https://chocolatey.org/install.ps1 C:\TEMP\choco-install.ps1
+
+# Let's be explicit about the shell that we're going to use.
+SHELL ["cmd", "/S", "/C"]
+
+# Install Build Tools. A 3010 error signals that requested operation is
+# successfull but changes will not be effective until the system is rebooted.
+RUN C:\TEMP\vs_buildtools.exe --quiet --wait --norestart --nocache `
+    --installPath C:\BuildTools `
+    --add Microsoft.VisualStudio.Workload.VCTools `
+    --add Microsoft.VisualStudio.Workload.MSBuildTools `
+    --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 `
+    --add Microsoft.VisualStudio.Component.Windows10SDK.17763 `
+ || IF "%ERRORLEVEL%"=="3010" EXIT 0
+
+RUN powershell C:\TEMP\choco-install.ps1
+RUN C:\TEMP\rustup-init.exe -y
```

Comment:
The Linux [Dockerfile](https://github.com/rust-vmm/rust-vmm-container/blob/master/Dockerfile) installs a specific version of the Rust toolchain, while this installs the latest. Would it make sense to pin the version here as well?

Reply:
Yes, it would certainly make sense to use the same version for both.

Reply:
IIRC, rust-init.exe did not allow requesting specific toolchain versions back then. I'll give the "--default-toolchain" argument a try before updating the PR.